File: 3 array/1 min value/minVal.cpp
// c++ program to find minimum value from array
#include <iostream>

int main()
{
    int dataArray[] = {36, 15, 7, 59, 36, 14, 25, 20, 78, 30, 45, 54, 23, 45, 67};

    int minValue = dataArray[0];

    for (int i : dataArray)
    {
        if (i < minValue)
        {
            minValue = i;
        }
    }

    std::cout << "minimum value is " << minValue;
    return 0;
}

File: 3 array/1 min value/minVal.py
# python program to find minimum value from array
data_array = [45,89,12,56,47,23,21,3,5,64,23,70,69,48]

min_val = data_array[0]

for i in data_array:
    if i < min_val:
        min_val = i

print(f"minimum value is {min_val}")